polygeist/tools/cgeist/Test/Verification/sycl/vec_add.cpp

```diff
@@ -1,5 +1,5 @@
-// RUN: clang++ -fsycl -fsycl-device-only -S -emit-mlir %s -o - | FileCheck %s --check-prefix=CHECK-MLIR
-// RUN: clang++ -fsycl -fsycl-device-only -fsycl-targets=spir64-unknown-unknown-syclmlir -S -emit-llvm %s -o - | FileCheck %s --check-prefix=CHECK-LLVM
+// RUN: clang++ -fsycl -fsycl-device-only -O0 -emit-mlir %s -o - | FileCheck %s --check-prefix=CHECK-MLIR
```

Comment:
why was `-S` dropped here?

Reply:
because is not needed, and I want all lit tests to be consistent.

Reply:
ok